File: app/src/main/java/com/example/hellofriend/Models/User1.java
package com.example.hellofriend.Models;

public class User1 {
        private String userId;
        private String name;
        private String email;
        private String profileImageUrl;

        public User1() {} // Empty constructor for Firestore

        public User1(String userId, String name, String email, String profileImageUrl) {
            this.userId = userId;
            this.name = name;
            this.email = email;
            this.profileImageUrl = profileImageUrl;
        }

        // Getters and setters
        public String getUserId() { return userId; }
        public void setUserId(String userId) { this.userId = userId; }

        public String getName() { return name; }
        public void setName(String name) { this.name = name; }

        public String getEmail() { return email; }
        public void setEmail(String email) { this.email = email; }

        public String getProfileImageUrl() { return profileImageUrl; }
        public void setProfileImageUrl(String profileImageUrl) { this.profileImageUrl = profileImageUrl; }
    }

File: app/src/main/java/com/example/hellofriend/Models/User1.kt
package com.example.hellofriend.Models

import com.google.firebase.Timestamp

class User1 {
    var userId: String? = null
    var name: String? = null
    var email: String? = null
    var contact: String? = null
    var gender: String? = null
    var address: String? = null
    var dateOfBirth: String? = null
    var userRole: String? = null
    var profileImageUrl: String? = null

    constructor() // Empty constructor for Firestore


    constructor(
        userId: String?,
        name: String?,
        email: String?,
        contact: String?,
        gender: String?,
        address: String?,
        dateOfBirth: String?,
        userRole: String?,
        profileImageUrl: String?
    ) {
        this.userId = userId
        this.name = name
        this.email = email
        this.contact = contact
        this.gender = gender
        this.address = address
        this.dateOfBirth = dateOfBirth
        this.userRole = userRole
        this.profileImageUrl = profileImageUrl
    }

}

